core/src/components/range/readme.md

```diff
@@ -76,30 +84,39 @@ left or right of the range.
+| `debounce`     | `debounce`      | How long, in milliseconds, to wait to trigger the `ionChange` event after each change in the range value.                                                                                                                                                              | `number`                                              | `0`         |
+| `disabled`     | `disabled`      | If `true`, the user cannot interact with the range.                                                                                                                                                                                                                    | `boolean`                                             | `false`     |
+| `dualKnobs`    | `dual-knobs`    | Show two knobs.                                                                                                                                                                                                                                                        | `boolean`                                             | `false`     |
+| `max`          | `max`           | Maximum integer value of the range.                                                                                                                                                                                                                                    | `number`                                              | `100`       |
+| `min`          | `min`           | Minimum integer value of the range.                                                                                                                                                                                                                                    | `number`                                              | `0`         |
+| `mode`         | `mode`          | The mode determines which platform styles to use.                                                                                                                                                                                                                      | `"ios" \| "md"`                                       | `undefined` |
+| `name`         | `name`          | The name of the control, which is submitted with the form data.                                                                                                                                                                                                        | `string`                                              | `''`        |
+| `neutralPoint` | `neutral-point` | The neutral point of the range slider. Default: value is `0` or the `min` when `neutralPoint < min` or `max` when `max < neutralPoint`.                                                                                                                                | `number`                                              | `0`         |
```

Comment:
So in the documentation here you say the neutral point will be `min` when `neutralPoint < min` but the case you check  is `min < neutralPoint` (https://github.com/ionic-team/ionic/pull/17400/files#diff-96cc2845be9d5c7024e04f2e1856b65fR57). Which is the correct one?

Reply:
nice catch 👍